### PR TITLE
Release GliaSDK 0.30.0

### PR DESCRIPTION
**Pull request by dukhovnyi**

### CODE REVIEW

**Review by gugalo — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,33 @@
...
+
+- Fixed media streaming on engagement transfer
+
+[0.30.0]: https://github.com/salemove/ios-sdk/compare/0.29.6...0.30.0
+
+## [0.29.6] - 2021-07-01
+
+### Added
+
+- Added possibility to start engegment with media
```
Comment (gersonnoboa) [inline CHANGELOG.md:25]: The support for this is incomplete in 0.29.6, so I think this has to be in 0.30.0, replacing the line about media direction.

Comment (dukhovnyi) [reply CHANGELOG.md]: fixed 21356ced6c2c62b1ff9bbe22f18afb2577e4cfb8

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,30 @@
...
 
-Unreleased: https://github.com/salemove/ios-sdk/compare/0.29.4...master
+Unreleased: https://github.com/salemove/ios-sdk/compare/0.30.0...master
+
+## [0.30.0] - 2021-07-26
+
+### Added
+
+- Added push notification support
+- Added possibility start engagement with media direction
```
Comment (gersonnoboa) [inline CHANGELOG.md:12]: I think this one is not needed anymore, as the `Added possibility to start engagement with media` covers it.

Comment (dukhovnyi) [reply CHANGELOG.md]: fixed cc9f93bf3c5d4fa196644a09f9d1b9e3c7430184

**Review by gersonnoboa — APPROVED** [review]

**Comment by dukhovnyi** [comment]
!merge

**Pull request merged**